resolwe_bio/processes/expression/htseq_count.yml

```diff
@@ -190,27 +190,32 @@
 
 
 - slug: htseq-count-raw
-  name: HTSeq-count-raw
+  name: HTSeq-count with counts per million normalization
   requirements:
     expression-engine: jinja
     executor:
       docker:
         image: resolwebio/rnaseq:3.0.0
   data_name: "{{ alignments.bam.file|basename|default('?') }} ({{ (alignments|sample_name) }})"
-  version: 1.2.2
-  type: data:expression:htseq:raw
+  version: 1.3.0
+  type: data:expression:htseq:cpm
   category: analyses
   flow_collection: sample
   persistence: CACHED
   description: >
-    Count the number of reads that map to a genomic feature (e.g. gene). For computationally
-    efficient quantification consider using featureCounts instead of HTSeq-count.
+    Count the number of reads that map to a genomic feature (e.g. gene)
+    and produce expressions according to the input parameters. The expressions
+    with raw counts, produced by HTSeq are then normalized by computing CPM.\n\n
+    See [the official website](https://htseq.readthedocs.io/en/release_0.9.1)
+    and [the introductory paper](https://academic.oup.com/bioinformatics/article/31/2/166/2366196)
+    for more information.\n\nFor computationally efficient quantification
+    consider using featureCounts instead of HTSeq-count.
   input:
     - name: alignments
       label: Aligned reads
       type: data:alignment:bam
-    - name: gff
-      label: Annotation (GFF)
+    - name: gtf
```

Comment:
Replace GFF with GTF in description of input fields [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/expression/htseq_count.yml#L246) and [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/expression/htseq_count.yml#L252).